frontend/src/assets/styles/themes/themeValues/colors.ts

```diff
@@ -1,9 +1,17 @@
+import { gold } from '@ant-design/colors';
+
 export type Colors = {
   primary: string;
   regularText: string;
+  gold: string;
+  gray: string;
```

Comment:
Może coś w tym stylu?
```suggestion
  activeStar: string;
  disactiveStar: string;
```

Reply:
zmienione